asserts/aspect_bundle.go

```diff
@@ -81,17 +82,23 @@ func assembleAspectBundle(assert assertionBase) (Assertion, error) {
 		return nil, err
 	}
 
-	v, ok := assert.headers["storage"]
-	if !ok {
-		return nil, fmt.Errorf(`"storage" stanza is mandatory`)
+	if assert.body == nil {
+		return nil, fmt.Errorf(`body must contain aspect schema`)
+	}
+
```

Comment:
do we need this check, as we know it will be done for use by assemble* ?

Reply:
We don't, I noticed the test coverage didn't reach it for that reason